### PR TITLE
Remove builds with PHP 5.5

### LINKED ISSUE #155: Docker nightly-5.5-* should be removed, as Prestashop now requires PHP >= 5.6
Nightly Docker images based on PHP 5.5 should probably be removed, as now PHP >= 5.6 is a requirement.

Tested with `nightly-5.5-apache`

### PR DESCRIPTION
**Pull request by Quetzacoalt91**
Fixes #155

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -14,13 +14,13 @@
...
 * Apache
-  * PHP 5.5: `1.5-5.5-apache`, `1.6-5.5-apache`, `1.7-5.5-apache`, `nightly-5.5-apache`
+  * PHP 5.5: `1.5-5.5-apache`, `1.6-5.5-apache`, `1.7-5.5-apache`
   * PHP 5.6: `1.5-5.6-apache`, `1.6-5.6-apache`, `1.7-5.6-apache`, `nightly-5.6-apache`
   * PHP 7.0: `1.5-7.0-apache`, `1.6-7.0-apache`, `1.7-7.0-apache`, `nightly-7.0-apache`
   * PHP 7.1: `1.5-7.1-apache`, `1.6-7.1-apache`, `1.7-7.1-apache`, `nightly-7.1-apache`
   * PHP 7.2: `1.5-7.2-apache`, `1.6-7.2-apache`, `1.7-7.2-apache`, `nightly-7.2-apache`
 * FPM
-  * PHP 5.5: `1.5-5.5-fpm`, `1.6-5.5-fpm`, `1.7-5.5-fpm`, `nightly-5.5-fpm`
+  * PHP 5.5: `1.5-5.5-fpm`, `1.6-5.5-fpm`, `1.7-5.5-fpm`
```
Comment (PierreRambaud) [inline README.md]:
```suggestion
  * PHP 5.5: `1.5-5.5-fpm`, `1.6-5.5-fpm`
```

File: README.md [hunk 2]
```diff
@@ -14,13 +14,13 @@
 ### Other PHP versions
 By default, our images are running with PHP 5.6. But each major version can be launched with another PHP version if you want to.
 * Apache
-  * PHP 5.5: `1.5-5.5-apache`, `1.6-5.5-apache`, `1.7-5.5-apache`, `nightly-5.5-apache`
+  * PHP 5.5: `1.5-5.5-apache`, `1.6-5.5-apache`, `1.7-5.5-apache`
```
Comment (PierreRambaud) [inline README.md]:
```suggestion
  * PHP 5.5: `1.5-5.5-apache`, `1.6-5.5-apache`
```

**Comment by Quetzacoalt91** [comment]
TO DO:

* Remove 1.7 with PHP 5.5 images
* Update script `generate_all.sh` to not create these removed items.

**Comment by Quetzacoalt91** [comment]
PR updated to remove ALL builds with PHP 5.5

**Pull request merged**